README.md

```diff
@@ -1,26 +1,44 @@
-# lambda-template
-A GitHub template for quickly starting a new AWS lambda project.
+# lambda-ec2-terminator
```

Comment:
this lambda is doing something a little more general than just terminating instances so this name seems misleading.  Would it make sense to change it to something like `lambda-ec2-state` or `lambda-ec2-state-setter`?

Reply:
I guess if I add more use cases that could make sense, but I'm not sure how useful it would be for Sage.